.gitignore

```diff
@@ -11,3 +11,4 @@ package-lock.json
 Pulumi.*.yaml
 .idea/
 *.iml
+key.rsa*
```

Comment:
Was this needed?  If so, perhaps keep the trailing newline?

Reply:
Not needed - but guides include generating the keys and I wanted to prevent anybody (me included) from committing them.
Sorry about the newline, my Windows soul still wins sometimes